### PR TITLE
Feature/add to cart

### PR DESCRIPTION
**Pull request by butschi**
@jacquelinend, @cloudyrainy, @jtsoi have a look …

If you would like a nicer HTML version of this, come talk to me.

### CODE REVIEW

**Comment by cloudyrainy** [comment]
@butschi looks good to me !

File: source/integration-guide/advanced.rst [hunk 1]
```diff
@@ -327,3 +327,68 @@ To enable the purchase history tooltip add the ``tooltipEnabled`` option to the
...
+
+Directly in the ``addWidget`` call::
+
+    vs.addWidget({
+        productId: "PRODUCT_ID",                        // Your product id you're using within Virtusize
+        buttonSelector: "BUTTON_SELECTOR",              // CSS selector for the virtusize button
+        availableSizes: ["small", "medium", "large"]    // Change this to the currently available sizes
+        done: function(error) {
+            this.on("user-added-item-to-cart", function(data) {
+                // Handle the event by added the appropiate item into the users
```
Comment (zyniq82) [inline source/integration-guide/advanced.rst]: Minor: Spelling: 
_Handle the event by **adding** the **appropriate** item into the **user's**_

Comment (zyniq82) [inline source/integration-guide/advanced.rst]: Same goes for the identical sentence in the other block below

Comment (jacquelinend) [inline source/integration-guide/advanced.rst]: Minor: remove comma in this sentence:
Therefore it is recommended that you integrate add to cart functionality only if you have dedicated product pages for every variant.

File: source/integration-guide/advanced.rst [hunk 2]
```diff
@@ -327,3 +327,68 @@ To enable the purchase history tooltip add the ``tooltipEnabled`` option to the
...
         tooltipStyle: "light"
     });
+
+
+Add to cart
+^^^^^^^^^^^
+You can enable an add to cart button directly inside the Virtusize widget.
+
+While it is not required, we recommend that you provide Virtusize with a list
+of **available sizes** when you enable add to cart. If you do so, the add to
```
Comment (jacquelinend) [inline source/integration-guide/advanced.rst]: Can you hyperlink "available sizes" to the documentation for setAvailableSizes(sizesList) in http://developers.virtusize.com/integration-guide/advanced.html ? (or elsewhere, if there is a better place to link to)

File: source/integration-guide/advanced.rst [hunk 3]
```diff
@@ -327,3 +327,68 @@ To enable the purchase history tooltip add the ``tooltipEnabled`` option to the
...
+cart button will be disabled and a little note will inform your customer for
```
Comment (jacquelinend) [inline source/integration-guide/advanced.rst]: reword - If you do so, for items that are out of stock, the add to cart button will be disabled and a note will inform your customer that the item is not available.

File: source/integration-guide/advanced.rst [hunk 4]
```diff
@@ -327,3 +327,68 @@ To enable the purchase history tooltip add the ``tooltipEnabled`` option to the
...
+items that are out of stock.
+
+.. note::
+
+    Virtusize does not handle color or variant selection at the moment.
+    Therefore it is recommended that you integrate add to cart functionality
+    only, if you have dedicated product pages for every variant. See
```
Comment (jacquelinend) [inline source/integration-guide/advanced.rst]: Minor: remove comma in this sentence:
Therefore it is recommended that you integrate add to cart functionality only if you have dedicated product pages for every variant.

**Comment by jacquelinend** [comment]
LGTM!

**Pull request merged**